src/Calculator.Shared/Views/Calculator.xaml

```diff
@@ -1559,9 +1559,10 @@
 				<PivotItem x:Name="HistoryPivotItem"
 										x:Uid="HistoryPivotItem"
 										Margin="0,10,0,0"
+										AutomationProperties.Name=""
 										AutomationProperties.AutomationId="HistoryLabel">
 					<PivotItem.Header>
-						<TextBlock x:Uid="HistoryLabel"
+						<TextBlock x:Uid="HistoryLabel" Text=""
```

Comment:
this shouldn't be necessary now (but `AutomationProperties.Name` above is still necessary)